ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -548,15 +559,15 @@ module('Acceptance | secrets/secret/create', function (hooks) {
     await logout.visit();
     await authPage.login(userToken);
     await settled();
-    await click(`[data-test-auth-backend-link=${enginePath}]`);
-
-    await click(`[data-test-secret-link="${secretPath}"]`);
-
+    await showPage.visit({ backend: enginePath, id: secretPath });
```

Comment:
For some reason the URL here is formatting incorrectly
<img width="467" alt="Screen Shot 2022-06-06 at 3 18 54 PM" src="https://user-images.githubusercontent.com/68122737/172258602-88a6a208-0ebe-40d3-a565-600588568081.png">

navigating directly to the URL seems to work, though
```suggestion
    await visit(`/vault/secrets/${enginePath}/show/${secretPath}`);
```